pkg/server/smtp/handler_test.go

```diff
@@ -85,14 +85,14 @@ func TestEmptyEnvelope(t *testing.T) {
 	// Test out some empty envelope without blanks
 	script := []scriptStep{
 		{"HELO localhost", 250},
-		{"MAIL FROM:<>", 501},
+		{"MAIL FROM:<>", 250},
```

Comment:
Could you add a comment along the lines of "empty FROM is valid SMTP," to remind me why we allow it given it looks weird, and prevent it from being removed in the future.